### DISCUSSION TITLE
Changelog clarification for absolute paths

### DISCUSSION THREAD

**Discussion by silverwind** [molefrog/wouter#413] [Q&A] [answered]
The changelog for 3.0.0 says:

> Route component can no longer use absolute paths. `<Route path="~/home" />` will no longer work.

This seems confusing because `~/home` is not an absolute path as per my understanding, but a relative one to the user's current home directory on a unix system. A absolute path in my book is one like used in the README examples: `/users/1`, e.g. it always starts with `/`.

**Comment by molefrog** [comment]
That's a good point, the term **absolute** is confusing. `~/home` refers to a top-level path, it allows bypassing any scoping introduced be nested routes as well as base path. Any ideas on how we can name this?

**Comment by silverwind** [comment]
Maybe "Route component can no longer use scope-bypassing absolute paths"?

**Reply by molefrog** [reply, marked as answer]
Sick! Fixed.

**Comment by molefrog** [comment]
> Maybe "Route component can no longer use scope-bypassing absolute paths"?
